### PR TITLE
Minor fixups

### PR DESCRIPTION
**Pull request by dweindl**

### CODE REVIEW

File: bin/petablint.py [hunk 1]
```diff
@@ -69,14 +69,11 @@ def parse_cli_args():
...
-             or not args.measurement_file_name):
-        # TODO: we should also allow running (limited) tests on a subset of
-        # files
-        parser.error('When not specifying a model name, sbml, '
-                     'condition and measurement file must be specified')
+    if not args.model_name and not args.sbml_file_name \
+        and not args.condition_file_name and not args.measurement_file_name \
+            and not args.parameter_file_name:
+        parser.error('Neither model name nor any filename specified. '
+                     'What shall I do?')
```
Comment (yannikschaelte) [inline bin/petablint.py]: :)

File: petab/core.py [hunk 2]
```diff
@@ -399,14 +401,17 @@ def get_optimization_to_simulation_parameter_mapping(
         The sbml model with observables and noise specified according to the
         petab format. Optional if par_sim_ids is provided.
 
-    par_opt_ids, par_sim_ids: list of str, optional
+    par_sim_ids: list of str, optional
         Ids of the optimization and simulation parameters. If not passed,
```
Comment (yannikschaelte) [inline petab/core.py]: Opt not needed any more? Adapt description

Comment (dweindl) [reply petab/core.py]: Was unused.

Comment (yannikschaelte) [inline petab/core.py]:
```suggestion
        Ids of the simulation parameters. If not passed,
```

**Pull request merged**